manager/allocator/cnmallocator/drivers_network_linux.go

```diff
@@ -26,3 +26,13 @@ func PredefinedNetworks() []networkallocator.PredefinedNetworkData {
 		{Name: "host", Driver: "host"},
 	}
 }
+
+// IsPredefinedNetwork checks if the network is a host/bridge network
+func IsPredefinedNetwork(target string) bool {
```

Comment:
One more thought: this method isn't really necessary because callers can just call  `PredefinedNetworks` and check against that list (with the same few lines of code that exist here). Let's remove this in favor of `PredefinedNetworks` to avoid interface bloat.